### PR TITLE
Added Vapor 3 clarity

### PR DESCRIPTION
**Pull request by Andrewangeta**
## Description
Added Vapor 3 specific usage

## Motivation and Context
People needed a Vapor 3 example

## How Has This Been Tested?
No need
## Checklist:
- [x] If applicable, I have updated the documentation accordingly.
- [x] If applicable, I have added tests to cover my changes.

### CODE REVIEW

**Comment by Joannis** [comment]
Do you think we'll need to show people that you access the DB by `let db = try request.make(MongoKitten.Database.self)` in the route?

**Comment by Andrewangeta** [comment]
Yea that makes sense. Adding now.

**Comment by Andrewangeta** [comment]
@Joannis Done :p

File: README.md [hunk 1]
```diff
@@ -54,15 +54,34 @@ import MongoKitten
...
+struct ServerLanguage: Content {
+   var language: String
+}
+
+func fetchTheBestServerLanguage(_ req: Request) throws -> EventLoopFuture<ServerLanguage> {
+    let db = try req.make(MongoKitten.Database.self)
+	
+    return db["server_languages"].findOne("language" == "swift").map { doucment in
+        guard let theBest = document else {
+            throw Abort(.internalServerError, reason: "Couldn't find the best, Node.JS is your future 🤢")
```
Comment (Joannis) [inline README.md]: Good example :) Maybe we can have a kitten/cat related joke here

Comment (Andrewangeta) [reply README.md]: Im open to suggestions.

**Pull request merged**